docs/concepts/gardenlet.md

```diff
@@ -510,6 +510,12 @@ Specifically, the following actions will be taken for the respective label and a
 * If the VPA has the annotation `eviction-requirements.autoscaling.gardener.cloud/downscale-restriction: never`, an `EvictionRequirement` is added to the VPA object that allows evictions for upscaling only
 * If the VPA has the annotation `eviction-requirements.autoscaling.gardener.cloud/downscale-restriction: in-maintenance-window-only`, the same `EvictionRequirement` is added to the VPA object when the Shoot is currently outside of its maintenance window. When the Shoot is inside its maintenance window, the `EvictionRequirement` is removed. Information about the Shoot maintenance window times are stored in the annotation `shoot.gardener.cloud/maintenance-window` on the VPA
 
+### [`WorkloadIdentityTokenRequestor` Controller](../../pkg/gardenlet/controller/workloadidentity/tokenrequestor)
```

Comment:
Do you plan to add more controllers to `pkg/gardenlet/controller/workloadidentity`?

Reply:
As of now - no. `workloadidentitytokenrequestor` seemed too long (and hard to read) for a dir name.

Reply:
Hm, true. OTOH, this is not following the directory structure. WDYT about introducing `pkg/gardenlet/controller/tokenrequestor/workloadidentity`, and `pkg/gardenlet/controller/tokenrequestor/secret`, and moving https://github.com/gardener/gardener/blob/ec4f782c99813450662dc3bdcb7fa50698d687e3/pkg/gardenlet/controller/add.go#L128-L136 into `pkg/gardenlet/controller/tokenrequestor/secret/add.go`?

Reply:
Yes, we can do that.

Reply:
TBH it still feels a little weird having token requestor in front. I think that `pkg/gardenlet/controller/tokenrequestor/secret` should be `pkg/gardenlet/controller/tokenrequestor/serviceaccount` if we stick to logic "the object for which the token is issued for". Let's see how the changes will look like, maybe we can iterate again if needed.

Reply:
I like it. Thank you :)